modules/ROOT/pages/rest-api-access.adoc

```diff
@@ -84,7 +84,7 @@ Note that the only role available for community-edition users is the *Full Admin
 
 h| Role
 h| Capability
-h| 7.02 DP
+h| 7.02
```

Comment:
Should this be 7.0.2 rather than 7.02? Also perhaps worth saying something like "7.0.2 and up" (just to head off the inevitable "what about 7.1" questions).

Reply:
Yup, good point.